client/src/modules/user/AuthBox/components/PasswordRecoveryForm.jsx

```diff
@@ -1,15 +1,15 @@
 import React, { PureComponent } from 'react';
 import { FormattedMessage, injectIntl } from 'react-intl';
 import { withRouter } from 'react-router-dom';
-import validator from 'validator';
 import _flowRight from 'lodash/flowRight';
 import _debounce from 'lodash/debounce';
 import _trim from 'lodash/trim';
 
 import { RouterMatchPropType, IntlPropType } from 'common/constants/propTypes';
-import { updatePassword } from '../model/actions';
+import { updatePassword } from 'modules/user/model/actions';
 import ValidationInput from './ValidationInput';
 import PendingButton from 'common/components/PendingButton';
+import { validatePassword as _validatePassword } from 'common/utils/helpers';
```

Comment:
maybe instead of using `_` in name we could name it like: `validatePasswordStructure` or sth similar do see that this helper function is doing only that?